src/silx/app/view/test/test_plotselection.py

```diff
@@ -0,0 +1,234 @@
+import pytest
+from silx.gui import qt
+from silx.io.dictdump import dicttoh5
+import silx.io
+from silx.app.view.CustomPlotSelectionWindow import CustomPlotSelectionWindow
+
+
+@pytest.fixture
+def setupMimeData(tmpdir):
+    # Create HDF5 file with 1D dataset in tmpdir
+    filename = str(tmpdir.join("test.h5"))
+    dicttoh5({"x": [1, 2, 3], "y1": [1, 2], "y2": [4, 3, 2, 1], "y3": []}, filename)
+
+    # Create mime data
+    mime_data = qt.QMimeData()
+    url = silx.io.url.DataUrl(file_path=filename, data_path="/x")
+    mime_data.setData("application/x-silx-uri", url.path().encode())
+
+    return mime_data
+
+
+@pytest.fixture
+def setupInvalidMimeData(tmpdir):
+    # Create HDF5 file with 2D dataset in tmpdir
+    filename = str(tmpdir.join("test.h5"))
+    dicttoh5({"x": [[1, 2], [3, 4]], "y": [[1, 2], [3, 4]]}, filename)
+
+    # Create mime data
+    mime_data = qt.QMimeData()
+    url = silx.io.url.DataUrl(file_path=filename, data_path="/x")
+    mime_data.setData("application/x-silx-uri", url.path().encode())
+
+    return mime_data
+
+
+def testDrop(qapp, setupMimeData, qWidgetFactory):
+    window = qWidgetFactory(CustomPlotSelectionWindow)
+
+    mime_data = setupMimeData
+
+    # Create drag enter event
+    dragEnterEvent = qt.QDragEnterEvent(
+        qt.QPoint(0, 0), qt.Qt.CopyAction, mime_data, qt.Qt.LeftButton, qt.Qt.NoModifier
+    )
+    window.getPlot1D().dragEnterEvent(dragEnterEvent)
+    qapp.processEvents()
+
+    # Create drag move event
+    dragMoveEvent = qt.QDragMoveEvent(
+        qt.QPoint(50, 50),
+        qt.Qt.CopyAction,
+        mime_data,
+        qt.Qt.LeftButton,
+        qt.Qt.NoModifier,
+    )
+    window.getPlot1D().dragMoveEvent(dragMoveEvent)
+    qapp.processEvents()
+
+    # Create drop event
+    dropEvent = qt.QDropEvent(
+        qt.QPoint(50, 50),
```

Comment:
[QDropEvent](https://doc.qt.io/qt-6/qdropevent.html) is expecting a `QPointF` and not a `QPoint` as `Drag[Enter|Move]Event` :shrug: ... This makes the CI fails:

```suggestion
        qt.QPointF(50., 50.),
```